test/fixtures/blueprint/addon/legacy-app/merge/my-app-install-blueprint-different-than-name/config/ember-cli-update.json

```diff
@@ -0,0 +1,30 @@
+{
+  "schemaVersion": "1.0.0",
+  "packages": [
+    {
+      "name": "ember-cli",
+      "version": "3.11.0-beta.1",
+      "blueprints": [
+        {
+          "name": "app",
+          "outputRepo": "https://github.com/ember-cli/ember-new-output",
+          "codemodsSource": "ember-app-codemods-manifest@1",
+          "isBaseBlueprint": true,
+          "options": [
+            "--no-welcome"
+          ]
+        }
+      ]
+    },
+    {
+      "name": "default-blueprint-different-than-name",
+      "location": "../legacy-blueprint",
+      "version": "0.0.1",
+      "blueprints": [
+        {
+          "name": "custom-blueprint"
```

Comment:
After the initial install, will someone be able to update this blueprint?

Reply:
or is it correct that after the initial resolution of the blueprint name from `install`, we no longer have to worry about it being a name mismatch (and we never have to look it up again)?

Reply:
The latter is correct. Once the blueprint name is set in `ember-cli-update.json` it will be the one generated when user wants to update the package version the blueprint belongs to.